articles/10-powerful-bi-tools-for-data-analysts/index.md

```diff
@@ -0,0 +1,218 @@
+* Users do not need professional expertise to use this tool.
+* It allows users to import data from various sources.
+  
+![IBM Cognos Analytics](/engineering-education/business-intelligence-tools-for-data-analysts/ibm-cognos-analytics.jpg)
+
+[Image Source: Value Coders](https://www.valuecoders.com/blog/wp-content/uploads/2020/09/ibm-cognos-business-intelligence-tools-2.jpg)
+
+#### 9. Yellowfin BI
+
+This tool offers functionalities such as mobile BI, mapping, collaborative BI, data discovery, storyboards, and dashboards. 
+
+##### Key features
+
+* It has intuitive filtering that uses radio buttons, sliders, and checkboxes.
+* Users can access the dashboard through mobile phones or web pages.
+* In this tool, users can share insights through simple scripts.
+* Its dashboards are AI-driven.
+* It provides BI solutions such as external apps for analytics, enterprise analytics, and embedded analytics. 
+
+![Yellowfin BI](/engineering-education/business-intelligence-tools-for-data-analysts/yellow-fin-bi.png)
+
+[Image Source: Weebly](http://yfcs.weebly.com/uploads/4/3/9/7/43974277/3215752_orig.png)
+  
+#### 10. QlikView
+
+This is a BI tool that enhances the safety and accuracy of data. This tool supports data aggregation and compression. 
+
+##### Key features
+
+* Users can analyze data and gain enhanced insights using its self-service feature.
+* It provides data visualization and presentation systematically.
+* It enables users from various sectors to get customized solutions. Some of the sectors include insurance, banking, and retail.
+* It has an associative engine feature that enables users to discover data.
+* In this tool, users can interact with analytics, dynamic apps, and dashboards.
+
+![QlikView](/engineering-education/business-intelligence-tools-for-data-analysts/qlikview.jpg)
+
+[Image Source: Xax](https://www.xax.de/wp-content/uploads/2018/11/financial-1024x726.jpg)
+  
+### Important Factors to Consider When Choosing a BI Tool
+
+* **Features:** Data analysts should look for BI tools that suit their needs. The features provided by these tools should meet the objectives of the business. Some of the important features to look at include customizable dashboards, data filters, data discovery, and automated reports. 
+* **User Interface:** If you are a data analyst, you would possibly want to use a BI tool that is user-friendly. The user interface should have a good design. It should also have clear displays that can enhance navigation within the tool.
+* **Ease of use:** How easy is it to use the BI tool? Good BI tools should offer tutorials and technical support to new users. 
+* **Price:** Employing business intelligence should be done cost-effectively. Businesses should choose a BI tool offering a price that is within its reach. 
+* **Integration:** Many companies want to gather data from diverse applications. Companies should choose BI Tools that are integrated with a wide variety of applications. 
+  

```

Comment:
A final summary (table) that discusses the various features and compares and contrasts the differences between the various BI tools is absolutely a great addition to this article.

Please discuss the pricing options of the various BI tools too.

For example,
![image](https://user-images.githubusercontent.com/34317373/102232511-39b5a200-3f15-11eb-9a93-600bf8a1116c.png)

Reply:
Thank you for this suggestion. I have created a table showing additional criteria for comparison (in addition to the features discussed in the article). This is because there are 10 BI tools, and each of them has many features. Putting all these features in one table would increase the size of the table. Please check. I am open to corrections.

Reply:
> Thank you for this suggestion. I have created a table showing additional criteria for comparison (in addition to the features discussed in the article). This is because there are 10 BI tools, and each of them has many features. Putting all these features in one table would increase the size of the table. Please check. I am open to corrections.

I just saw the table that you added in the latest commit. Really good work! It gave me a crisp summary of the BI Tools.